slycot/transform.py

```diff
@@ -333,6 +333,254 @@ def tb04ad(n,m,p,A,B,C,D,tol1=0.0,tol2=0.0,ldwork=None):
     kdcoef = max(index)+1
     return A[:Nr,:Nr],B[:Nr,:m],C[:p,:Nr],Nr,index,dcoeff[:porm,:kdcoef],ucoeff[:porm,:porp,:kdcoef]
 
+
+def tb05ad(n, m, p, jomega, A, B, C, job='NG'):
+    """At, Bt, Ct, g_jw, rcond, ev, hinvb = tb05ad_a(n, m, p, jomega, A, B, C):
```

Comment:
This doc string isn't right: what is returned depends on job (which isn't listed as an argument).

I'm not sure if we (Slycot and python-control) have a good convention for different return forms.  Numpy appears to just omit the return list in the header (e.g., [numpy.linalg.svd](https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.svd.html)) and give details below.

`tb05ad_a` => `tb05ad`.